src/cloudant/database.py

```diff
@@ -18,9 +18,17 @@
 import json
 import contextlib
 import posixpath
-import urllib
+
 from requests.exceptions import HTTPError
 
+# pylint: disable=wrong-import-order
+from ._2to3 import PY2
+if PY2:
+    # pylint: disable=wrong-import-order,no-name-in-module
+    from urllib import quote_plus
+else:
+    from urllib.parse import quote_plus  # pylint: disable=import-error,no-name-in-module
```

Comment:
That does not work with pylint 1.6/python 2.7, or at least, I could not get it to work.  pylint would only pass with the override on the same line (a "line" override) vs. above (a "block" override).  The preferred "block" style override works fine with Python 3.5.  I noted this in prior comments -- if you know how to make it work I'm happy to try.  I've moved all PY2 conditionals to _2to3 based on Dave's feedback so this exact instance is gone at least.